src/main/java/codeu/model/data/Message.java

```diff
@@ -67,4 +70,10 @@ public String getContent() {
   public Instant getCreationTime() {
     return creation;
   }
+
+  public String getMarkdownContent() {return markdown(content);}
+  private String markdown(String s) {Parser parser = Parser.builder().build();
```

Comment:
Not a big deal, but it would look nicer if that statement was on its own line haha